app/uploaders/avatar_uploader.rb

```diff
@@ -39,7 +39,7 @@ def store_dir
 
   # Override the filename of the uploaded files:
   # Avoid using model.id or version_name here, see uploader/store.rb for details.
-  # def filename
-  #   "something.jpg" if original_filename
-  # end
+  def filename
+    "#{Time.now}.png" if original_filename.present?
```

Comment:
But what will happen if several people upload avatar at the same time? Files will have the same name, it is not good